lib/Stow.pm.in

```diff
@@ -377,6 +378,13 @@ sub stow_contents {
         next NODE if $node eq '..';
         my $node_target = join_paths($target, $node);
         next NODE if $self->ignore($stow_path, $package, $node_target);
+
+	if ($self->{dotfiles}) {
```

Comment:
Please use spaces to indent, not tabs, to stay consistent with the rest of the code base.